src/forms.css

```diff
@@ -126,11 +130,11 @@
  */
 .input:disabled,
 .textarea:disabled,
-.select:disabled {
+.select--stroke:disabled {
   pointer-events: none;
   color: var(--darken50);
   background-color: var(--disabled-interactive-color);
-  box-shadow: inset 0 0 0 1px var(--disabled-interactive-color-dark);
+  box-shadow: inset 0 0 0 1px var(--disabled-interactive-color-dark) !important;
```

Comment:
Is this `!important` required?

Reply:
Without the `!important`, the disabled `select--stroke` elements retain their colored borders (see below). I wonder if there's a better way to achieve the desired effect without using `!important`? I thought using it might make sense given the context -- that is, we always want disabled `select--stroke`s to use the disabled color variable.

<img width="760" alt="Screen Shot 2021-04-14 at 2 28 29 PM" src="https://user-images.githubusercontent.com/15935667/114781871-22a18800-9d2e-11eb-80cb-4f0e005ecd40.png">

Reply:
makes sense to keep !important